### PR TITLE
First stab at home page.

### PR DESCRIPTION
**Pull request by drhodrid**
Putting up a pull request. At this stage:
1. Things need formatting by @angus-g if possible.
2. The animation (uploaded to you tube) needs to be sized and positioned appropriately.
3. I'd like thoughts on the content. What should be added/removed.

### CODE REVIEW

**Comment by angus-g** [comment]
I've just (force) pushed to address points 1 and 2. As for 3, I think it feels a bit wordy for a short, sharp introduction. Maybe more of a focus on what it *does*, rather than what it *uses*?

Also, I don't know where the source for the G-ADOPT logo is, but I think it could look better on a white (transparent) background, and add an outline on the ice and ocean at the top, so they pop out a little more.

**Comment by angus-g** [comment]
I've just added a sub-section of the homepage with the "About" section that contains the original more verbose blurb about the components. We can drop that and just merge the homepage for the moment if that needs a bit more time.

**Comment by drhodrid** [comment]
I’ll do this now!

On 21 Feb 2024, at 11:51 am, Angus Gibson ***@***.***> wrote:



I've just added a sub-section of the homepage with the "About" section that contains the original more verbose blurb about the components. We can drop that and just merge the homepage for the moment if that needs a bit more time.

**Comment by drhodrid** [comment]
I'm now happy with this as a first stab. I think I will update the page when all other sections are in place, providing directions on how to use the webpage etc....

**Review by angus-g — APPROVED** [review]
Okay, I think we'll push this up as it is now!
